village/src/components/SmurfForm.js

```diff
@@ -1,30 +1,50 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 
 class SmurfForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: '',
-      age: '',
-      height: ''
+      
+        name: '',
+        age: '',
+        height: ''
+      
     };
   }
 
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    let smurf = {name: this.state.name, age: this.state.age, height: this.state.height}
+  axios.post("http://localhost:3333/smurfs", smurf)
```

Comment:
The axios call needs to be in `App.js` because the list of smurfs we want to update lives in the App component's state.
```suggestion
  addSmurf = event => {
    event.preventDefault();
    this.props.addSmurf(this.state);
    this.setState({
      name: "",
      age: "",
      height: ""
    });
    this.props.history.push("/");
  };
```